src/inmanta/graphql/schema.py

```diff
@@ -0,0 +1,117 @@
+"""
+Copyright 2025 Inmanta
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+Contact: code@inmanta.com
+"""
+
+import typing
+
+import inmanta.data.sqlalchemy as models
+import strawberry
+from inmanta.data import get_session, get_session_factory
+from sqlalchemy import Select, asc, desc, select
+from strawberry import relay
+from strawberry.schema.config import StrawberryConfig
+from strawberry.types import Info
+from strawberry.types.info import ContextType
+from strawberry_sqlalchemy_mapper import StrawberrySQLAlchemyLoader, StrawberrySQLAlchemyMapper
+
+mapper: StrawberrySQLAlchemyMapper[typing.Any] = StrawberrySQLAlchemyMapper()
+SCHEMA: strawberry.Schema | None = None
+
+
+def get_expert_mode(root: "Environment") -> bool:
+    if not hasattr(root, "settings"):
+        return False
+    return bool(root.settings.get("enable_lsm_expert_mode", False))
+
+
+@mapper.type(models.Project)
+class Project:
+    pass
+
+
+@mapper.type(models.Environment)
+class Environment:
+    is_expert_mode: bool = strawberry.field(resolver=get_expert_mode)
+
+
+def get_schema() -> strawberry.Schema:
+    if SCHEMA is None:
+        initialize_schema()
+    assert SCHEMA
+    return SCHEMA
```

Comment:
Why is this here? i.e. why use a global with an intializer method instead of declaring it at the module level?